api/src/main/java/io/strimzi/plugin/security/profiles/ContainerSecurityProviderContext.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.plugin.security.profiles;
+
+import io.fabric8.kubernetes.api.model.SecurityContext;
+import io.strimzi.api.kafka.model.storage.Storage;
+
+public interface ContainerSecurityProviderContext {
```

Comment:
Can you add some Javadocs?

Reply:
And possibly a `package-info.java` to summarize the purpose of the package (since it's in `api`).